datanode/gateway/graphql/schema.graphql

```diff
@@ -858,6 +858,9 @@ type Query {
     pagination: Pagination
   ): TransferConnection
 
+  "Find a transfer using its ID"
```

Comment:
bit of a nit pick but should this be GET as per: https://github.com/vegaprotocol/vega/blob/develop/DOCUMENTATION_STYLE.md#naming-terminology

I know its FIND has been used below on line 864